src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import classNames from "classnames";
+
+import style from "./Loading.module.scss";
+
+export enum LoaderSize {
+  s = "s",
+  m = "m",
+  l = "l",
+}
+
+export type LoaderProps = {
+  loading?: boolean;
+  size?: LoaderSize;
+  className?: string;
+};
+
+const Loader: React.FC<LoaderProps> = ({
+  loading = true,
+  size = LoaderSize.m,
+  className = "",
+}) => {
+  let loaderClasses = classNames("style.loader", `style.loader_size-${size}`);
+
+  loaderClasses = className ? loaderClasses + " " + className : "";
```

Comment:
какая-то сложная логика, достаточно просто прокинуть в функцию `className`, если там что-то будет, то она вставит, нет - не вставит
```suggestion
  let loaderClasses = classNames("style.loader", `style.loader_size-${size}`, className);
```